test/model-db-test.js

```diff
@@ -0,0 +1,34 @@
+const expect = require('chai').expect;
+const sinon = require('sinon');
+const db = require('../src/models');
+const { User } = db;
+const winston = require('winston');
+
+const logger = winston.createLogger({
+  level: 'info',
+  format: winston.format.json(),
+  transports: [
+    //
+    // - Write all logs error (and below) to `error.log`.
+    //
+    new winston.transports.File({
+      filename: '/Users/kjeter/Sites/info.log',

```

Comment:
Should be written to a /logs folder in the project directory, which should also be .gitignored.